packages/tools/devtools/devtools-browser-extension/package.json

```diff
@@ -80,15 +82,20 @@
 		"@fluidframework/map": "workspace:~",
 		"@fluidframework/mocha-test-setup": "workspace:~",
 		"@fluidframework/sequence": "workspace:~",
+		"@fluidframework/test-utils": "workspace:~",
 		"@fluidframework/tinylicious-client": "workspace:~",
 		"@microsoft/api-extractor": "^7.34.4",
 		"@types/chai": "^4.0.0",
 		"@types/chrome": "0.0.232",
+		"@types/expect-puppeteer": "2.2.1",
```

Comment:
I assume at some point we were importing `expect-puppeteer`. If we won't anymore, this can go as well?

Reply:
Removed

Reply:
Actually we need it

Reply:
@chentong7 what is it that needs this? We aren't using `expect-puppeteer`, so the corresponding types package shouldn't be needed.

Reply:
The expect-puppeteer library has global definition of page, browser that we will use in test cases.